packages/react-reconciler/src/__tests__/__snapshots__/ReactIncrementalPerf-test.internal.js.snap

```diff
@@ -357,8 +357,8 @@ exports[`ReactDebugFiberPerf skips parents during setState 1`] = `
 
 ⚛ (Committing Changes)
   ⚛ (Committing Snapshot Effects: 0 Total)
-  ⚛ (Committing Host Effects: 6 Total)
-  ⚛ (Calling Lifecycle Methods: 6 Total)
```

Comment:
Should have spotted this earlier. Sigh.

Reply:
Is this just way more efficient now?

Reply:
I think not — I think it was still overcounting actual updates (not entirely sure why). But it's better.